src/components/VirtualList/Config.ts

```diff
@@ -116,24 +116,24 @@ const istioType: ResourceType<IstioConfigItem> = {
 };
 
 export const IstioTypes = {
-  gateway: { name: 'Gateway', icon: 'G' },
-  virtualservice: { name: 'VirtualService', icon: 'VS' },
-  destinationrule: { name: 'DestinationRule', icon: 'DR' },
-  serviceentry: { name: 'ServiceEntry', icon: 'SE' },
-  rule: { name: 'Rule', icon: 'R' },
-  adapter: { name: 'Adapter', icon: 'A' },
-  template: { name: 'Template', icon: 'T' },
-  quotaspec: { name: 'QuotaSpec', icon: 'QS' },
-  quotaspecbinding: { name: 'QuotaSpecBinding', icon: 'QSB' },
-  policy: { name: 'Policy', icon: 'P' },
-  meshpolicy: { name: 'MeshPolicy', icon: 'MP' },
-  servicemeshpolicy: { name: 'ServiceMeshPolicy', icon: 'SMP' },
-  clusterrbacconfig: { name: 'ClusterRbacConfig', icon: 'CRC' },
-  rbacconfig: { name: 'RbacConfig', icon: 'RC' },
-  servicemeshrbacconfig: { name: 'ServiceMeshRbacConfig', icon: 'SRC' },
-  sidecar: { name: 'Sidecar', icon: 'S' },
-  servicerole: { name: 'ServiceRole', icon: 'SR' },
-  servicerolebinding: { name: 'ServiceRoleBinding', icon: 'SRB' }
+  gateway: { name: 'Gateway', slug: 'gateways', icon: 'G' },
+  virtualservice: { name: 'VirtualService', slug: 'virtualservices', icon: 'VS' },
+  destinationrule: { name: 'DestinationRule', slug: 'destinationrules', icon: 'DR' },
+  serviceentry: { name: 'ServiceEntry', slug: 'serviceentries', icon: 'SE' },
+  rule: { name: 'Rule', slug: 'rules', icon: 'R' },
+  adapter: { name: 'Adapter', slug: 'adapters', icon: 'A' },
```

Comment:
Commented before: templates/adapters are special as we supported legacy formats (probably from Istio 1.4 we can refactor this).